src/scope/AccessToken.js

```diff
@@ -0,0 +1,96 @@
+import Scope from './Scope'
+import Account from '../entity/Account'
+import ADIOrder from '../entity/ADIOrder'
+import Product from '../entity/Product'
+import Thng from '../entity/Thng'
+import Collection from '../entity/Collection'
+import Action from '../entity/Action'
+import ActionType from '../entity/ActionType'
+import Project from '../entity/Project'
+import PurchaseOrder from '../entity/PurchaseOrder'
+import Redirector from '../entity/Redirector'
+import ShipmentNotice from '../entity/ShipmentNotice'
+import Place from '../entity/Place'
+import File from '../entity/File'
+import OpeatorAccess from '../entity/OperatorAccess'
```

Comment:
`OperatorAccess`, `ADIOrderEvent`, `CommissionState`, `Application`, `Domain`, `Property`, `ReactorLog`, `ReactorSchedule`, `ReactorScript`, `Redirection`, and `ShortDomain` are all sub-resources of other entities, similar to Operator, so not needed here.

Reply:
fixed, thanks